src/devices/CpuTemperature/README.md

```diff
@@ -5,6 +5,11 @@ Device bindings for the CPU Temperature Sensor
 ## Summary
 
 Returns the current temperature of the CPU Temperature Sensor. Useful telemetry in its own right, but also useful for calibrating the Raspberry Pi Sense HAT.
+  
+Note: No hardware needs to connected for this to work. It gets the CPU core temperature, typically about 40 degress Celcius.  
+On a Raspberry Pi. it does not work with Windows 10 IoT-Core; Raspian is the required operating system. 
```

Comment:
```suggestion
The binding requires a Raspberry Pi running Raspbian. It won't work if the Raspberry Pi's OS is Windows 10 IoT-Core. 
```

Reply:
The binding requires any Linux (possibly Unix will work as well), Raspberry PI is not required.